application/src/test/resources/fixtures/config.yaml

```diff
@@ -7,6 +7,9 @@ ses:
   endpoint: test.email.com
   region: test-region-2
   from-address: test@sanctionco.com
+  success-html: fixtures/success-fixture.html
```

Comment:
When you change the `JsonProperty` to camel case, make sure you update all of the config files.